cob_default_robot_config/cob4-1/command_gui_buttons.yaml

```diff
@@ -32,6 +32,21 @@ group17:
   component_name: behavior
   buttons: [[torso_front,trigger,torso_front],
     [torso_front_home,trigger,torso_front_home]]
+group18:
+  group_name: head
+  component_name: head
+  buttons: [[stop,trigger,stop],
+    [init,trigger,init],
+    [recover,trigger,recover],
+    [home,move,home],
+    [front,move,front],
```

Comment:
no, they are not
see also my comment in https://github.com/ipa320/cob_robots/pull/410#issue-131418376